qiskit/circuit/bit.py

```diff
@@ -59,53 +59,45 @@ def __init__(self, register=None, index=None):
             self._repr = f"{self.__class__.__name__}({self._register}, {self._index})"
 
     @property
+    @deprecate_func(
+        is_property=True,
+        since="0.17",
+        additional_msg=(
+            "Instead, use the method ``QuantumCircuit.find_bit`` to find all the containing "
```

Comment:
can we sphinx crossref?

(I'll not type out this comment again if it appears elsewhere)